### PR TITLE
Change default lp from 30303 to c_defaultListenPort

### PR DESCRIPTION
**Pull request by ijcoe6ru**

### CODE REVIEW

File: libp2p/Network.h [hunk 1]
```diff
@@ -17,6 +17,7 @@
 /** @file Network.h
  * @author Alex Leverington <nessence@gmail.com>
  * @author Gav Wood <i@gavwood.com>
+ * @author ijcoe6ru <s6yg1ez3@mail2tor.com>
```
Comment (chfast) [inline libp2p/Network.h:20]: @chriseth @bobsummerwill I guess this is misleading.

Comment (bobsummerwill) [inline libp2p/Network.h:20]: Yes, @ijcoe6ru.   We are only using author tags for substantial contributions (if at all).
Much easier to keep these globally.    Who has touched particular files is way too granular to be useful.

**Comment by bobsummerwill** [comment]
Beyond the author tag, though, looks-good-to-me.

**Comment by chfast** [comment]
Why closed?

**Comment by bobsummerwill** [comment]
He made a new one, and that was merged as https://github.com/ethereum/libweb3core/pull/84.

**Pull request closed without merging**